splice_machine/src/test/java/com/splicemachine/derby/security/PermissionIT.java

```diff
@@ -0,0 +1,169 @@
+package com.splicemachine.derby.security;
+
+import com.splicemachine.db.shared.common.reference.SQLState;
+import com.splicemachine.derby.test.framework.*;
+import com.splicemachine.test.HBaseTest;
+import com.splicemachine.test.SerialTest;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+
+import java.sql.ResultSet;
+
+import static com.splicemachine.derby.test.framework.SpliceUnitTest.assertFailed;
+import static com.splicemachine.derby.test.framework.SpliceUnitTest.resultSetSize;
+import static java.lang.String.format;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by yxia on 6/22/18.
+ */
+@Category(value = {SerialTest.class,HBaseTest.class})
```

Comment:
Does it have to be Serial?

Reply:
Yes, since the two test cases testCachedSchemaPrivilege() and testCachedTablePrivilege() work on the same set of schema/tables and each of them have grant/revoke privileges operations. In case they run in parallel, the permission check result may vary.